Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: build 11.6 10.2
```

Comment:
I don't know much about Makefiles but is it necessary to have `build 11.6 10.2` instead of just `build` here?